docs/application/dotnet/guides/wcircularui/overview.md

```diff
@@ -0,0 +1,94 @@
+# Wearable Circular UI
+
+The Wearable Circular UI project is a set of helpful extensions of the <a href="https://docs.microsoft.com/en-us/xamarin/xamarin-forms/" target="_blank">Xamarin.Forms</a> framework.
+The aim of this project is to develop an open source software motivate software developer to creating Tizen Wearable Xamarin Forms application more easily and efficiently.<br>
+The binaries are available via NuGet. The package name is <a href="https://github.com/Samsung/Tizen.CircularUI/" target="_blank">Tizen.Wearable.CircularUI</a>.
+
+> **Note**
+>
+>Xamarin Forms provides cross-platform APIs, but this project are only worked on the Samsung Gear device that support Tizen .NET.
```

Comment:
Xamarin.Forms provides cross-platform APIs, but this project works only on the Samsung Gear device that 
supports Tizen .NET.

Reply:
done